### PR TITLE
Export 2020 01 09

### PR DESCRIPTION
**Pull request by Conjuror**

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -1446,6 +1446,45 @@ msgctxt "logo_text"
...
+msgctxt "recommended_news_title"
+msgid "Recommended News For You"
+msgstr ""
+
+#, c-format
+msgctxt "recommended_news_content"
+msgid ""
+"Tap “Yes” to begin seeing news recommended for you by our AI news engine."
+" You can turn off recommendations and control your settings anytime in "
+"News-&gt;Settings. %1$s"
```
Comment (Delphine) [inline locales/templates/app.pot]: The placeholder should be explained in the localization comment.

File: locales/templates/app.pot [hunk 2]
```diff
@@ -1446,6 +1446,45 @@ msgctxt "logo_text"
...
+msgstr ""
+
+#. Recommended news choose language
+msgctxt "recommended_news_preference_language"
+msgid ""
+"Choose Your Prefered\n"
```
Comment (Delphine) [inline locales/templates/app.pot]: Should be "preferred".
Also, what does "preferred" refer to? Should be explained in localization comment so translators know how to  localize it.

File: locales/templates/app.pot [hunk 3]
```diff
@@ -2384,3 +2423,48 @@ msgctxt "travel_search_engine_fxlite"
...
+msgstr ""
+
+#. dialog description for introducing the feature. The %1$s here is Firefox
+#. Lite.
+#, c-format
+msgctxt "travel_dialog_1_description"
+msgid "Explore things to do, accomodations, and more in %1$s Travel Discovery."
+msgstr ""
+
+#. Action button for seeing the search result in Firefox Lite Travel Discovery
```
Comment (Delphine) [inline locales/templates/app.pot]: The comment doesn't seem to correspond to the string, unless I'm missing something?

File: locales/templates/app.pot [hunk 4]
```diff
@@ -2384,3 +2423,48 @@ msgctxt "travel_search_engine_fxlite"
...
+msgctxt "travel_dialog_1_action_1"
+msgid "TRY TRAVEL DISCOVERY"
+msgstr ""
+
+#. Action button for dismissing this prompt
+msgctxt "travel_dialog_1_action_2"
+msgid "NO, THANKS"
+msgstr ""
+
+#. dialog title for asking if users want to change default search engine
```
Comment (Delphine) [inline locales/templates/app.pot]: Comment doesn't correspond to  string

**Comment by Delphine** [comment]
Some comments seemed a bit off. 
Also, seems like there is already a conflict 🙈

File: locales/templates/app.pot (hunk 1 above)
Comment (youwenliang) [reply locales/templates/app.pot]: @Conjuror 
Change
#. Recommended news onboarding title and content.
to
#. Recommended news onboarding title and content. The %1$s here is a placeholder for "Learn more" which directs users to SUMO page with detail information about the feature.

File: locales/templates/app.pot (hunk 2 above)
Comment (youwenliang) [reply locales/templates/app.pot]: @Conjuror 
Change 
#. Recommended news choose language
to
#. Recommended news choose your preferred language

and Change
"Choose Your Prefered\n"
to
"Choose Your Preferred\n"

File: locales/templates/app.pot (hunk 3 above)
Comment (youwenliang) [reply locales/templates/app.pot]: This is a dialog (prompt) for first time users. I think the comment makes sense to me.

I do see an error here though,
need to change "accomodations" to "accommodations"
@Conjuror

File: locales/templates/app.pot (hunk 4 above)
Comment (youwenliang) [reply locales/templates/app.pot]: Do you mean "Action button for dismissing this prompt"?
The prompt here referred to the dialog mentioned above, so one action button is to try travel discovery, and the other action button is to dismiss the prompt, does that make sense?

File: locales/templates/app.pot (hunk 1 above)
Comment (tinahsieh) [reply locales/templates/app.pot]: DONE

File: locales/templates/app.pot (hunk 2 above)
Comment (tinahsieh) [reply locales/templates/app.pot]: DONE

File: locales/templates/app.pot (hunk 4 above)
Comment (tinahsieh) [reply locales/templates/app.pot]: I rewrote the comment and hope it becomes clearer for you.

File: locales/templates/app.pot (hunk 3 above)
Comment (tinahsieh) [reply locales/templates/app.pot]: DONE

File: locales/templates/app.pot (hunk 2 above)
Comment (tinahsieh) [reply locales/templates/app.pot]: DONE

File: locales/templates/app.pot [hunk 5]
```diff
@@ -1446,6 +1446,47 @@ msgctxt "logo_text"
 msgid "Service provided by"
 msgstr ""
 
+#. Recommended news onboarding title and content. The %1$s here is a
+#. placeholder for "Learn more" which directs users to SUMO page with detail
+#. information about the feature.
+msgctxt "recommended_news_title"
+msgid "Recommended News For You"
```
Comment (Delphine) [inline locales/templates/app.pot]: I don't see the placeholder in the string anymore, am I missing something?

File: locales/templates/app.pot [hunk 6]
```diff
@@ -1446,6 +1446,47 @@ msgctxt "logo_text"
...
+msgstr ""
+
+#, c-format
+msgctxt "recommended_news_content"
+msgid ""
+"Tap “Yes” to begin seeing news recommended for you by our AI news engine."
+" You can turn off recommendations and control your settings anytime in "
+"News-&gt;Settings. %1$s"
```
Comment (Delphine) [inline locales/templates/app.pot]: I still don't see the placeholder explained in the string here.

**Comment by Conjuror** [comment]
Weird. Let me check my side.

File: locales/templates/app.pot (hunk 1 above)
[the remark this answers is not in the document]
Comment (Conjuror) [reply locales/templates/app.pot]: check w/ UX now. Will fix it

File: locales/templates/app.pot (hunk 2 above)
[the remark this answers is not in the document]
Comment (Conjuror) [reply locales/templates/app.pot]: Weird. It's Preferred here.
https://github.com/mozilla-l10n/zerda-android-l10n/pull/62/files#diff-439a3673577e31c15de38af37aeac43eR1515

File: locales/templates/app.pot (hunk 5 above)
Comment (Conjuror) [reply locales/templates/app.pot]: I guess they messed up with previous one. Will check with UX

**Comment by Conjuror** [comment]
Updated. Please help on reviewing it. Thank you @Delphine

**Comment by Delphine** [comment]
Looks good now, but there is a conflict now in the meantime, unfortunately.
Should be the last time hopefully :)

**Pull request merged**